src/lib.rs

```diff
@@ -258,6 +275,33 @@ pub fn decode_image(
     Ok(buffer)
 }
 
+#[cfg(feature = "pixbuf")]
+/// Decodes the given blurhash to an [Pixbuf][gdk_pixbuf::Pixbuf] of the specified size.
+///
+/// The punch parameter can be used to de- or increase the contrast of the
+/// resulting image.
+pub fn decode_pixbuf(
+    blurhash: &str,
+    width: u32,
+    height: u32,
+    punch: f32,
+) -> Result<gdk_pixbuf::Pixbuf, Error> {
+    use std::convert::TryInto;
+    let bytes = decode(blurhash, width, height, punch)?;
+    let width = width.try_into().unwrap_or_default();
+    let height = height.try_into().unwrap_or_default();
```

Comment:
```suggestion
        let width = width.try_into().expect("width fits in i32");
        let height = height.try_into().expect("height fits in i32");
```

Wouldn't fall back to default for these, since 0 really implies an error condition.